pkg/util/validation/limits.go

```diff
@@ -38,6 +38,7 @@ type Limits struct {
 	CreationGracePeriod       time.Duration       `yaml:"creation_grace_period"`
 	EnforceMetadataMetricName bool                `yaml:"enforce_metadata_metric_name"`
 	EnforceMetricName         bool                `yaml:"enforce_metric_name"`
+	EnforceLabelNameCharset   bool                `yaml:enforce_label_name_charset"`
```

Comment:
Do we want to expose this option to Cortex users? Or is it supposed to be only internal flag used by extension you’re writing? (I would prefer later, but if there is good reason to expose, we can keep it)

Reply:
Same concern here. I would prefer to avoid making it configurable to Cortex users unless there's a legit use case for that.

Reply:
Ok, no problem, i'll remove the flag so that the setting isn't exposed to the user. I'll add a parameter to `validation.ValidateLabels()` then, to skip the label name validation. I'll also extend the unit tests to test that flag